packages/style/src/provider/provider-browser.tsx

```diff
@@ -0,0 +1,37 @@
+import { ProviderComponent, UseCacheHook } from './types';
+
+if (typeof window === 'undefined') {
+  throw new Error(
+    `
+ ██████╗ ██████╗ ███╗   ███╗██████╗ ██╗██╗     ███████╗██████╗
```

Comment:
you definitely have to extract this as a constant.

Reply:
ehehee

yeah i'll make sure this won't affect bundle size outside of dev. will put a process.env infront of it.